versions.yaml

```diff
@@ -0,0 +1,43 @@
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/go-sql-driver/mysql/splunkmysql
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/graph-gophers/graphql-go/splunkgraphql
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/jackc/pgx/splunkpgx
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/jinzhu/gorm/splunkgorm
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/jmoiron/sqlx/splunksqlx
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/julienschmidt/httprouter/splunkhttprouter
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/lib/pq/splunkpq
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/miekg/dns/splunkdns
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb
+      - github.com/signalfx/splunk-otel-go/instrumentation/github.com/tidwall/buntdb/splunkbuntdb
+      - github.com/signalfx/splunk-otel-go/instrumentation/gopkg.in/olivere/elastic/splunkelastic
+      - github.com/signalfx/splunk-otel-go/instrumentation/internal
+      - github.com/signalfx/splunk-otel-go/instrumentation/k8s.io/client-go/splunkclient-go
+      - github.com/signalfx/splunk-otel-go/instrumentation/net/http/splunkhttp
+
+excluded-modules:
+  - github.com/signalfx/splunk-otel-go/build
+  - github.com/signalfx/splunk-otel-go/example
+  - github.com/signalfx/splunk-otel-go/instrumentation/database/sql/splunksql/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/go-sql-driver/mysql/splunkmysql/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/confluentinc/confluent-kafka-go/kafka/splunkkafka/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/go-chi/chi/splunkchi/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/gomodule/redigo/splunkredigo/redis/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/go-sql-driver/mysql/splunkmysql/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/graph-gophers/graphql-go/splunkgraphql/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/jackc/pgx/splunkpgx/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/julienschmidt/httprouter/splunkhttprouter/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/lib/pq/splunkpq/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/miekg/dns/splunkdns/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/github.com/tidwall/buntdb/splunkbuntdb/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/gopkg.in/olivere/elastic/splunkelastic/test
+  - github.com/signalfx/splunk-otel-go/instrumentation/k8s.io/client-go/splunkclient-go/transport/test
```

Comment:
I added these to not create and push unnecessary tags.

Reply:
Addressed https://github.com/signalfx/splunk-otel-go/pull/1989/commits/144b9fc9f103d7b9f856065797bfe87ecf519c7a